### ISSUE TITLE
ROVA exception

### ISSUE THREAD

**Issue by mkrijtenberg** [pippyn/Home-Assistant-Sensor-Afvalbeheer#235]
Hi,

I love this addon, and I use it warn me when to put the bins on the street. However, it stopped working a few days ago, and I missed a pickup day for PMT, and now I have wait another month haha :(

This is the log message I get:

```
Traceback (most recent call last):
File "/config/custom_components/afvalbeheer/sensor.py", line 438, in async_update
await self.collector.update()
File "/config/custom_components/afvalbeheer/sensor.py", line 1223, in update
response = json.loads(r.text)
File "/usr/local/lib/python3.8/json/__init__.py", line 357, in loads
return _default_decoder.decode(s)
File "/usr/local/lib/python3.8/json/decoder.py", line 337, in decode
obj, end = self.raw_decode(s, idx=_w(s, 0).end())
File "/usr/local/lib/python3.8/json/decoder.py", line 355, in raw_decode
raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

I guess it doesn't get the correct json data anymore....

Also, my config:
```yaml
sensor:
    # ROVA
  - platform: afvalbeheer
    wastecollector: rova
    resources:
      - gft
      - papier
      - pmd
    postcode: ****
    streetnumber: **
    upcomingsensor: 1
    dateonly: 1
```

**Comment by swttt** [comment]
Same issue here.

I am not 100% sure, but it looks like they changed their website? (and possibly api?)
The website looks more like afvalwijzer now, and i am not sure if it was like that in the past.... (allways used their app, that didnt change by the way)

**Comment by Devqon** [comment]
Same here, it stopped working a few days ago

**Comment by mkrijtenberg** [comment]
Maybe someone can fix this, but for now, I'm switching over to: https://github.com/xirixiz/homeassistant-afvalwijzer
Their ROVA implementation (still) works.

**Comment by pippyn** [comment]
I'm aware of this issue. A fix will be released soon! Sorry for the delay.

**Comment by pippyn** [comment]
Fixed in v4.7.22

**Comment by Devqon** [comment]
Just updated to 4.7.22 and it's working again, thanks @pippyn !
![image](https://user-images.githubusercontent.com/9316480/120322629-e974a580-c2e4-11eb-967c-2957d2e3cd4b.png)
